cookbooks/cdo-secrets/recipes/globals.rb

```diff
@@ -1,4 +1,5 @@
-template "/home/#{node[:current_user]}/#{node.chef_environment}/globals.yml" do
+template 'globals' do
```

Comment:
yup, exactly- I shortened the name here just so I didn't need to duplicate the whole path when referencing the resource above.